pkg/k8shandler/common.go

```diff
@@ -364,7 +364,7 @@ func newPodTemplateSpec(nodeName, clusterName, namespace string, node api.Elasti
 	selectors := mergeSelectors(node.NodeSelector, commonSpec.NodeSelector)
 	// We want to make sure the pod ends up allocated on linux node. Thus we make sure the
 	// linux node selectors is always present. See LOG-411
-	selectors = ensureLinuxNodeSelector(selectors)
+	selectors = EnsureLinuxNodeSelector(selectors)
```

Comment:
> this doesn't need to be publicly available.. any particular reason for changing this?

Yes.  Most of the `indemanagement` bits were added to a package which doesn't have visibility to k8shandler.  I intentionally did this because `k8shandler` is monolithic and needs to be refactored to provide better structure IMO.  This allows resusing a function that could/should exist elsewhere